src/moveTo.js

```diff
@@ -121,7 +121,7 @@ const MoveTo = (() => {
    * @param  {object} options Custom options
    */
   MoveTo.prototype.move = function(target, options = {}) {
-    if (target !== 0 && !target) {
```

Comment:
`0` is **falsy** in JavaScript. So you need not check that if you are checking for `!target`. 

All these evaluate to false.
`if (false),
if (null),
if (undefined),
if (0),
if (NaN),
if (''),
if (""),
if (document.all) [1]`